lib/util/git/index.ts

```diff
@@ -696,15 +697,15 @@ async function handleCommitAuth(localDir: string): Promise<void> {
   await writeGitAuthor();
 }
 
-export async function commitFiles({
+export async function prepareCommit({
   branchName,
   files,
   message,
   force = false,
-}: CommitFilesConfig): Promise<CommitSha | null> {
+}: CommitFilesConfig): Promise<CommitResult | null> {
   const { localDir } = GlobalConfig.get();
   await syncGit();
-  logger.debug(`Committing files to branch ${branchName}`);
+  logger.debug(`Preparing files for commiting to branch ${branchName}`);
```

Comment:
I'd expect one more 't' in the last verb. Also see https://www.spellchecker.net/misspellings/commiting

Reply:
Good point, thanks. I'll fix it with an upcoming PR